e2e/scripts/validate-generated-project.js

```diff
@@ -39,9 +45,9 @@ const validateExtensibilityConfig = async (project, templateVersion) => {
     const pkg = require(pkgPath)
     return new Promise((resolve, reject) => {
         if (
-            !pkg.hasOwnProperty('ccExtensibility') ||
-            !pkg['ccExtensibility'].hasOwnProperty('extends') ||
-            !pkg['ccExtensibility'].hasOwnProperty('overridesDir') ||
+            !Object.prototype.hasOwnProperty.call(pkg, 'ccExtensibility') ||
```

Comment:
Why do we need to change this?

Reply:
So apparantly we're not supposed to call hasOwnProperty directly on an object because a certain property or method on the object might be overridden and is not a safe check.
Now that we enabled linting on this directory, it complains about the same.

So a safer way is to call `Object.prototype.hasOwnProperty.call(pkg, 'ccExtensibility')` or even better or recommended way is to now use `hasOwn()` instead of `hasOwnProperty`. I updated it to use `hasOwn` to avoid invoking `call()`